Snippets/Snippets_4/DelayedDelivery/DeferForTimeSpan.cs

```diff
@@ -8,6 +8,9 @@ class DeferForTimeSpan
         public void SendDelayedMessage()
         {
             IBus bus = null;
+            #region configure-timeout-manager
+            // Timeout Manager is part of NServiceBus Core, no need to call anything.
```

Comment:
@SimonCropp would you just not have this snippet in v4 and add a note in the text explaining why it is missing?